tests/utils/test_databricks_utils.py

```diff
@@ -200,3 +202,15 @@ def test_databricks_params_throws_errors(ProfileConfigProvider):
     ProfileConfigProvider.return_value = mock_provider
     with pytest.raises(Exception):
         databricks_utils.get_databricks_host_creds()
+
+
+def test_is_in_databricks_runtime():
+    with mock.patch("sys.modules", new={**sys.modules, "pyspark.databricks": mock.MagicMock()}):
+        # pylint: disable=unused-import,import-error,no-name-in-module,unused-variable
+        import pyspark.databricks  # noqa
```

Comment:
Can we remove `noqa`?

Reply:
remove noqa then still print "unused-variable" lint error. Not sure why.

Reply:
Is the error emitted from pylint or pycodestyle?

Reply:
seems my fault. we can remove noqa